woocommerce/woocommerce-ios
# Remove unused code on `AvailabilityAction`

Related to #7055 


### Description


While working on https://github.com/woocommerce/woocommerce-ios/pull/7622, there was a change on the unused code  `AvailabilityStore`. I thought deleting all code in the same PR was distracting, so I opened a separate PR to delete this unused code.

### Testing instructions


Feel free to do a confidence check to make sure the dashboard works as before.



---
